src/Storefront/Resources/views/storefront/component/checkout/offcanvas-cart-summary.html.twig

```diff
@@ -21,19 +21,20 @@
             {% set activeShipping = page.cart.deliveries.elements[0] %}
 
             {% block component_offcanvas_summary_content_info %}
-                <div class="row offcanvas-shipping-info">
-                    <span class="col-7 shipping-label shipping-cost">
-                        <strong>{{ "checkout.summaryShipping"|trans|sw_sanitize }}</strong>
+                {% for shipping in page.cart.deliveries %}
+                    <div class="row offcanvas-shipping-info">
```

Comment:
a block within the for loop would be nice

and maybe some more blocks around the other texts :blush:

Reply:
hi
jap gerne, no problem :)

Reply:
i've added a few blocks, ok like this @mitelg ?